app/assets/stylesheets/thredded/components/_post.scss

```diff
@@ -78,7 +78,7 @@
     @extend %thredded--blockquote;
   }
 
-  .spoiler {
+  .thredded--post--content--spoiler {
```

Comment:
Since this is a fully qualified class, it no longer needs to be nested, so: `&--spoiler`